src/ui/TimeSpanPicker.tsx

```diff
@@ -95,6 +82,12 @@ const TimeSpanPicker = ({
   const { t } = useTranslation();
   const idPrefix = `${id}-time-span-picker`;
 
+  const filterTimes = useCallback(
+    (time: string) =>
+      isQuarterHour(time) && timeToNumeric(time) >= timeToNumeric(startTime),
+    [startTime],
+  );
+
```

Comment:
I think the value '23:59' now won't be included in the endTime picker

Reply:
I wasn't sure if that was still needed, what was the original use case for it? Since the user can still type 23:59 if need be

Reply:
That is the initial value of the endTime when creating an event. It indicates till the end of the day, so it would be weird if you can't select it via the picker imo. It makes it harder to enter full day time values.

Reply:
Ah yes ok it's for full day events makes sense, will add it back